lib/utils.ts

```
@@ -94,34 +95,44 @@ export const generateFormSchema = (questions: FormQuestion[]) => {
+              ])
+              .transform((val) => {
+                // Ensure we always return a string
+                if (typeof val !== 'string') {
+                  console.warn(
+                    `Field ${curr.name} transformed non-string to empty string:`,
+                    val
+                  );
+                  return '';
+                }
+                return val;
+              });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)